milatools/cli/commands.py

```diff
@@ -81,478 +95,778 @@ def main():
-        if search:
-            terms = "+".join(search)
-            url = f"https://sites.google.com/search/mila.quebec/mila-intranet?query={terms}&scope=site&showTabs=false"
-        else:
-            url = "https://intranet.mila.quebec"
-        print(f"Opening the intranet: {url}")
-        webbrowser.open(url)
+    # ----- mila docs ------
 
-    def init():
-        """Set up your configuration and credentials."""
+    docs_parser = subparsers.add_parser(
+        "docs",
+        help="Open the Mila cluster documentation.",
+        formatter_class=SortingHelpFormatter,
+    )
+    docs_parser.add_argument("SEARCH", nargs=argparse.REMAINDER, help="Search terms")
+    docs_parser.set_defaults(function=docs)
 
-        #############################
-        # Step 1: SSH Configuration #
-        #############################
+    # ----- mila intranet ------
 
-        print("Checking ssh config")
+    intranet_parser = subparsers.add_parser(
+        "intranet", help="Open the Mila intranet in a browser."
+    )
+    intranet_parser.add_argument(
+        "SEARCH", nargs=argparse.REMAINDER, help="Search terms"
+    )
+    intranet_parser.set_defaults(function=intranet)
 
-        setup_ssh_config()
-        # TODO: Move the rest of this command to functions in the init_command module,
-        # so they can more easily be tested.
+    # ----- mila init ------
 
-        print("# OK")
+    init_parser = subparsers.add_parser(
+        "init",
+        help="Set up your configuration and credentials.",
+        formatter_class=SortingHelpFormatter,
+    )
+    init_parser.set_defaults(function=init)
 
-        #############################
-        # Step 2: Passwordless auth #
-        #############################
+    # ----- mila forward ------
 
-        print("Checking passwordless authentication")
+    forward_parser = subparsers.add_parser(
+        "forward",
+        help="Forward a port on a compute node to your local machine.",
+        formatter_class=SortingHelpFormatter,
+    )
+    forward_parser.add_argument("REMOTE", help="node:port to forward")
+    forward_parser.add_argument(
+        "--page",
+        help="String to append after the URL",
+        default=None,
+        metavar="VALUE",
+    )
+    forward_parser.add_argument(
+        "--port",
+        type=int,
+        help="Port to open on the local machine",
+        default=None,
+        metavar="VALUE",
+    )
+    forward_parser.set_defaults(function=forward)
+
+    # ----- mila code ------
 
-        here = Local()
+    code_parser = subparsers.add_parser(
+        "code",
+        help="Open a remote VSCode session on a compute node.",
+        formatter_class=SortingHelpFormatter,
+    )
+    code_parser.add_argument(
+        "PATH", help="Path to open on the remote machine", type=str
+    )
+    code_parser.add_argument(
+        "--alloc",
+        nargs=argparse.REMAINDER,
+        help="Extra options to pass to slurm",
+        metavar="VALUE",
```

Comment:
```suggestion
        metavar="VALUE",
        default=[],
```

This is expanded in a statement that looks like `[..., *alloc]` later so it's important to make sure it is not None.